src/main/java/qna/domain/Answer.java

```diff
@@ -43,15 +47,23 @@ public Answer(Long id, User writer, Question question, String contents) {
         this.contents = contents;
     }
 
-    public Answer setDeleted(boolean deleted) {
-        this.deleted = deleted;
-        return this;
+    public ContentType getContentType() {
+        return contentType;
+    }
+
+    public DeleteHistory deleteSelf(LocalDateTime deleteDate) {
+        setDeleted(true);
+        return new DeleteHistory(contentType, getId(), writer, deleteDate);
     }
 
     public boolean isDeleted() {
         return deleted;
     }
 
+    public void setDeleted(boolean deleted) {
```

Comment:
다음과 같이 객체의 멤버를 변환하는 메서드는 외부에서 접근하지 못하도록 최소화 하시면 좋을 것 같아요!
```suggestion
    private void setDeleted(boolean deleted) {
```

Reply:
.... 눈이 삐었나 봅니다. private인줄 알았네요. 수정하겠습니다.